mattermost_giphy/app.py

```diff
@@ -1,19 +1,25 @@
 # -*- coding: utf-8 -*-
-import logging
-import os
-import sys
+
 import json
-from urlparse import urlsplit
-from urlparse import urlunsplit
-import re
+import logging
 import random
+import re
+
+try:
+    from urllib.parse import urlsplit
+    from urllib.parse import urlunsplit
+except ImportError:
+    from urlparse import urlsplit
+    from urlparse import urlunsplit
 
 import requests
 from flask import Flask
-from flask import request
 from flask import Response
+from flask import request
 
-from mattermost_giphy.settings import *
+from mattermost_giphy.settings import (
+        USERNAME, ICON_URL, RATING, SCHEME, GIPHY_API_KEY,
+        MATTERMOST_GIPHY_TOKEN)
```

Comment:
it would look better to put the closing parenthesis on a new line

Reply:
if we want to address coding style, let's do it once and for all with CI, but that's not the purpose of this PR I guess

Reply:
hence me saying LGTM before, as well as putting a single comment and not a review :wink:

Reply:
<3